Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[workspace]
+
+members = [
+  "dummy",
+]
+resolver = "3"
+
+[workspace.package]
+version = "0.1.0"
```

Comment:
A discussion to have at some point: do we use the same version number for all crates or a version per crate?